web/src/upload.js

```diff
@@ -261,6 +261,9 @@ export function setup_upload(config) {
         event.preventDefault();
         const files = event.originalEvent.dataTransfer.files;
         upload_files(uppy, config, files);
+        if (config.mode === "compose" && !compose_state.composing()) {
+            compose_actions.start("stream");
+        }
```

Comment:
What's the thinking on this being before the `upload_files` call? Also, I think probably we want to start a reply; this logic seems like it'll open a stream compose box if you're in a direct message thread?

Reply:
Ooh, this bug went un-noticed, will update it.

Reply:
> What's the thinking on this being before the `upload_files` call?

I just added it in last, But you are right we should call before file upload begins.



> Also, I think probably we want to start a reply; this logic seems like it'll open a stream compose box if you're in a direct message thread?

Ooh, this bug went un-noticed, will update it.

Reply:
@timabbott , I have updated the commit, PTAL.

Reply:
Looks good, thanks!